pkg/asset/tls/variables.go

```diff
@@ -0,0 +1,228 @@
+package tls
+
+import (
+	"crypto/x509"
+	"crypto/x509/pkix"
+	"fmt"
+	"net"
+	"time"
+
+	"github.com/openshift/installer/pkg/types"
+)
+
+const (
+	validityTenYears      = time.Hour * 24 * 365 * 10
+	validityThirtyMinutes = time.Minute * 30
+)
+
+func init() {
+	RootCA := &CertKey{
+		IsRoot:       true,
+		Subject:      pkix.Name{CommonName: "root-ca", OrganizationalUnit: []string{"openshift"}},
+		KeyUsages:    x509.KeyUsageKeyEncipherment | x509.KeyUsageDigitalSignature | x509.KeyUsageCertSign,
+		Validity:     validityTenYears,
+		KeyFileName:  "root-ca.key",
+		CertFileName: "root-ca.crt",
+
+		IsCA: true,
+	}
+
+	KubeCA := &CertKey{
+		Subject:      pkix.Name{CommonName: "kube-ca", OrganizationalUnit: []string{"bootkube"}},
+		KeyUsages:    x509.KeyUsageKeyEncipherment | x509.KeyUsageDigitalSignature | x509.KeyUsageCertSign,
+		Validity:     validityTenYears,
+		KeyFileName:  "kube-ca.key",
+		CertFileName: "kube-ca.crt",
```

Comment:
I expect we can generate these filenames from the subject common name, in which case they don't have to be part of our `CertKey` config.  For an example in my mockup, see [here][1] (my `Asset`s have names, so getting the asset filename is a generic `Asset` method, not a TLS-specific method).

[1]: https://github.com/wking/openshift-installer/blob/30de9cdc79447b127ac5894543da2db524904385/installer/pkg/assets/asset.go#L122-L125